### PR TITLE
Delete items total and qty update in PO and PR

### PR DESCRIPTION
**Pull request by PITPL-Kaushal-Khute**
Delete Items Total and Total_Qty Update

### CODE REVIEW

File: erpnext/stock/doctype/purchase_receipt/purchase_receipt.js [hunk 1]
```diff
@@ -110,6 +110,33 @@ frappe.ui.form.on("Purchase Receipt", {
...
+		var base_total = 0.0;
+		$.each(frm.doc.items || [], function (i, row) {
+			if (row.qty) {
+				total_qty += flt(row.qty);
+			}
+			row.total_qty = flt(row.qty) - flt(row.total_qty);
+			frm.refresh_field("total_qty", total_qty);
+		});
+		frm.set_value("total_qty", Math.abs(total_qty));
+		console.log("total_allocated_amount 3", total_qty)
```
Comment (PITPL-Akshay-Prajapati) [inline erpnext/stock/doctype/purchase_receipt/purchase_receipt.js]: remove console

File: erpnext/buying/doctype/purchase_order/purchase_order.js [hunk 2]
```diff
@@ -84,6 +84,33 @@ frappe.ui.form.on("Purchase Order", {
...
+		var base_total = 0.0;
+		$.each(frm.doc.items || [], function (i, row) {
+			if (row.qty) {
+				total_qty += flt(row.qty);
+			}
+			row.total_qty = flt(row.qty) - flt(row.total_qty);
+			frm.refresh_field("total_qty", total_qty);
+		});
+		frm.set_value("total_qty", Math.abs(total_qty));
+		console.log("total_allocated_amount 3", total_qty)
```
Comment (PITPL-Akshay-Prajapati) [inline erpnext/buying/doctype/purchase_order/purchase_order.js]: remove console

**Review by PITPL-Akshay-Prajapati — CHANGES_REQUESTED** [review]

File: erpnext/stock/doctype/purchase_receipt/purchase_receipt.js [hunk 3]
```diff
@@ -110,6 +110,33 @@ frappe.ui.form.on("Purchase Receipt", {
...
+		var total = 0.0;
+		var base_total = 0.0;
+		$.each(frm.doc.items || [], function (i, row) {
+			if (row.amount) {
+				total += flt(row.amount);
+				base_total += flt(flt(row.amount) * flt(row.exchange_rate));
+			}
+		});
+		frm.set_value("total", Math.abs(total));
+		console.log("total_allocated_amount", total)
```
Comment (PITPL-Akshay-Prajapati) [inline erpnext/stock/doctype/purchase_receipt/purchase_receipt.js]: remove console

File: erpnext/buying/doctype/purchase_order/purchase_order.js [hunk 4]
```diff
@@ -84,6 +84,33 @@ frappe.ui.form.on("Purchase Order", {
...
+		var total = 0.0;
+		var base_total = 0.0;
+		$.each(frm.doc.items || [], function (i, row) {
+			if (row.amount) {
+				total += flt(row.amount);
+				base_total += flt(flt(row.amount) * flt(row.exchange_rate));
+			}
+		});
+		frm.set_value("total", Math.abs(total));
+		console.log("total_allocated_amount", total)
```
Comment (PITPL-Akshay-Prajapati) [inline erpnext/buying/doctype/purchase_order/purchase_order.js]: remove console

**Review by PITPL-Akshay-Prajapati — CHANGES_REQUESTED** [review]

File: erpnext/stock/doctype/purchase_receipt/purchase_receipt.js (hunk 1 above)
Comment (PITPL-Kaushal-Khute) [reply erpnext/stock/doctype/purchase_receipt/purchase_receipt.js]: Done

**Review by PITPL-Akshay-Prajapati — APPROVED** [review]

**Pull request merged**